### DISCUSSION TITLE
Debugging access policies?

### DISCUSSION THREAD

**Discussion by divan** [geldata/gel#5285] [Q&A]
Is there better way to debug access policies than commenting them out in schema and migrating?

I have strange behavior of set of multiple access policies that I can't understand (I'm sure I've made a mistake somewhere). It would be helpful to enable/disable particular policies, run queries and see the output. At the moment the only way I found to do this is via commenting out and running a migrations. Is there any better way?

**Comment by Sporarum** [comment]
I agree, it's especially complicated to debug things like `CardinalityViolationError`, since we can't see what is visible and what is hidden, and by which access policy
